src/app/Actions/user.js

```diff
@@ -0,0 +1,15 @@
+import * as type from '../Constants/ActionTypes';
+
+export function addFriend(id) {
+  return {
+    type: types.ADD_FRIEND,
+    id
+  };
+};
+
+export function removeFriend(id) {
```

Comment:
we can pass both actions and data (if needed to reducers) because reducers can listen for the actions and pick up their action types in switch blocks